custom-configurations/src/main/java/com/sequenceiq/cloudbreak/domain/CustomConfigurations.java

```diff
@@ -0,0 +1,140 @@
+package com.sequenceiq.cloudbreak.domain;
+
+import java.io.Serializable;
+import java.util.Set;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+import javax.persistence.UniqueConstraint;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+@Entity
+@Table(
+        uniqueConstraints = @UniqueConstraint(columnNames = {"name", "crn"})
```

Comment:
I'm not sure this is what you would expect. This means the name and crn together should be unique. From the SQL script it seems to me that crn alone should be unique, also name + account has to be unique

Reply:
Yes, name and account should be unique and crn must be unique on its own, will update the entity class.

Reply:
Updated